tidb-cloud/migration-job.md

```diff
@@ -0,0 +1,185 @@
+
+## Step 2: Configure the source and target connection
+
+On the **Create Migration Job** page, configure the source and target connection.
+
+1. Enter a job name, which must start with a letter and must be less than 60 characters. Letters (A-Z, a-z), numbers (0-9), underscores (_), and hyphens (-) are acceptable.
+
+2. Fill in the source connection profile.
+
+   - **Data source**: the data source type.
+   - **Region**: the region of the data source, which is required for cloud databases only.
+   - **Connectivity method**: the connectivity method for the data source. Currently, public IP, VPC Peering, or Private Private Link is supported.
+   - **Hostname or IP address** (for public IP and VPC Peering): the hostname or IP address of the data source.
+   - **Service Name** (for Private Link): the endpoint service name.
+   - **Port**: the port of the data source.
+   - **Username**: the username of the data source.
+   - **Password**: the password of the username.
+   - **SSL/TLS**: whether to use SSL/TLS connection for the data source. If you enable SSL/TLS, you need to upload the certificates of the data source, including the CA certificate, client certificate, and client key.
+
+3. Fill in the target connection profile.
+
+   - **Username**: enter the username of the target cluster in TiDB Cloud.
+   - **Password**: enter the password of the TiDB Cloud username.
+
+4. Click **Validate Connection and Next** to validate the information you have entered.
+
+5. Take action according to the message you see:
+
+    - If you use Public IP or  VPC Peering, you need to add the Data Migration service's IP addresses to the IP Access List of your source database and firewall (if any).
+    - If you use Private Link, you are prompted to accept the endpoint request in your account.
+
+## Step 3: Choose the objects to be migrated
+
+1. Choose full data migration, incremental data migration, or both by choosing the checkboxes.
+
+    > **Tip:**
+    >
+    > - To migrate data to TiDB Cloud once and for all, choose both full data migration and incremental data migration, which ensures data consistency between the source and target databases.
+    > - To migrate the existing data of the source database to TiDB Cloud, choose the full data migration checkbox.
+
+2. On the **Choose Objects to Migrate** page, select the objects to be migrated. You can click **All** to select all objects, or click **Customize** and then click the checkbox next to the object name to select the object.
+
+    - If you click **All**, the migration job will migrate the existing data from the whole source database instance to TiDB Cloud and replicate ongoing changes after the full migration.
+
+    ![Select All Objects](/media/tidb-cloud/migration-job-select-all.png)
+
+    - If you click **Customize** and select some databases, the migration job will migrate the existing data and replicate ongoing changes of the selected databases to TiDB Cloud.
```

Comment:
If only the full data migration checkbox is selected in the previous step, clicking **Customize** will not replicate ongoing changes after the full migration.